downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc

```diff
@@ -3,11 +3,10 @@
 
 = Deploying your system for container signing
 
-Execution Environments are container images used by Ansible {ControllerName} to run jobs. This content can be downloaded to the private automation hub, and published within your organization.
+{HubNameStart} implements image signing so you can rely on better security for the EE container images.
```

Comment:
I don't believe EE is an approved acronym and the appropriate attribute should be used in this instance.